docs/user_guide/queue_config.md

```diff
@@ -395,6 +395,7 @@ There is no guaranteed resource setting for the root queue.
 
 Maximum resources when configured place a hard limit on the size of all allocations that can be assigned to a queue at any point in time.
 A maximum resource can be set to 0 which makes the resource not available to the queue.
+However, setting all maximum resources to zero will be ignored and logged for debugging.
```

Comment:
Change from:
> However, setting all maximum resources to zero will be ignored and logged for debugging.

to:
However, a maximum resource with all resource types set to 0 will be ignored, not set on the queue, and logged for debugging.

Reply:
I just noticed that setting all the maximum resource to 0 does not simply get ignored. 
If a maximum resource setting already exists in the queue, the previous setting will be cleared.

This behavior is not documented too. 

https://github.com/apache/yunikorn-core/blob/ff654b2dd9038c1b026d99e553f79cd32e3d4de7/pkg/scheduler/objects/queue.go#L407-L408

Reply:
ah yes that is correct, that was introduced via YUNIKORN-2169 to allow removing a quota. if the queue is configured and no longer has a quota.
Good catch. We need to add that behaviour as it was a fix requested but never documented.